src/org/opendatakit/aggregate/parser/BaseFormParserForJavaRosa.java

```diff
@@ -111,12 +116,29 @@ private static final void initializeJavaRosa() {
              // new CoreModelModule().registerModule();
              // replace with direct call to PrototypeManager
              PrototypeManager.registerPrototypes(SERIALIABLE_CLASSES);
+             redirectOutput();
              new XFormsModule().registerModule();
              isJavaRosaInitialized = true;
        }
     }
   }
 
+  private static void redirectOutput() {
+    File jrLogFile = new File(FileSystemUtils.getBriefcaseFolder(), ".briefcase-javarosa.log");
+    try {
+      final PrintStream jrOut = new PrintStream(jrLogFile);
+      Std.setOut(jrOut);
+      Std.setErr(jrOut);
+      Runtime.getRuntime().addShutdownHook(new Thread() {
```

Comment:
I agree, it's not strictly necessary. 

> When I removed the hook and ran the application, the log was created.

That's weird. It should create the file anyway. I'm not sure why you mention these as a pair. The hook just ensures explicit close of the file (rather than relying on the OS to clean up the dangling file descriptors when killing the process).